AutomatedTesting/Gem/PythonTests/assetpipeline/fbx_tests/fbx_tests.py

```diff
@@ -747,7 +803,9 @@ def populate_asset_info(workspace, project, assets):
                                            + product.product_name
 
     @staticmethod
-    def compare_scene_debug_file(asset_processor, expected_file_path, actual_file_path):
+    def compare_scene_debug_file(asset_processor, expected_file_path, actual_file_path, expected_hashes_to_skip = None, actual_hashes_to_skip = None):
+        import re
```

Comment:
nit: avoid import in a method, it will make method take longer and is somewhat of a python anti-pattern unless you have a very considered reason (ie just for debugging)

Reply:
Gotcha, I'll change it. I thought it was preferred to import at the closest scope possible and to minimize imports at the file level.